templates/.reek.yml

```diff
@@ -13,6 +13,16 @@ directories:
     "app/helpers":
       UtilityFunction:
         enabled: false
+      FeatureEnvy:
```

Comment:
we think that helpers are meant exactly to extract some logic which could be also done in a model but don't belong there